collectors/python.d.plugin/adaptec_raid/README.md

```diff
@@ -1,12 +1,18 @@
-# adaptec raid
+# Adaptec RAID controller monitoring with Netdata
 
-Module collects logical and physical devices health metrics.
+Collects health metrics on logical and physical devices.
```

Comment:
Probably i dont translate it correctly, but the module collects _logical and physical devices metrics_, not _on logical and physical devices metrics_.